doc/installation.md

```diff
@@ -174,7 +174,9 @@ Simply run these commands in Windows PowerShell:
 ### macOS
 
 kmonad supports macOS 10.12 to 10.15 (Sierra, High Sierra, Mojave, and
-Catalina) and macOS 11.0 (Big Sur).
+Catalina) and macOS 11.0 (Big Sur). When using driverkit-based extension
+v2.1.0 and later, kmonad also supports macOS 16.0 (Ventura) and 17.0
+(Sonoma).
```

Comment:
```suggestion
v2.1.0 and later, kmonad also supports macOS 13.0 (Ventura) and 14.0
(Sonoma).
```

I think the version numbers might be mixed up with the recent iOS releases. 🤔

Reply:
Got it, it should be 13.0 and 14.0. I just fixed it. 👍